lib/helpers.js

```diff
@@ -123,7 +146,9 @@ module.exports = {
    */
   log(msg) {
     /* istanbul ignore next */
-    if (!global.isSilent) console.log(msg)
+    if (global.isSilent) return
+    if (global.isStdout) console.error(msg)
```

Comment:
Ok it seems to make everything more complicate, let's keep it simple and leave it as it is for now